services/web-app/pages/contributing/schema.mdx

```diff
@@ -1,17 +1,34 @@
-# Resource schemas
+---
+title: Schema
+description:
+  Carbon resources (e.g. libraries, assets, design kits) follow a schema to standardize definitions
+  and documentation. This structured information is indexed by Carbon's platform to use in the
+  website and other services.
+---
 
-Carbon Design System resources (e.g. libraries, assets, design kits) follow a schema to document
-each resource in a standardized way. This structured information is indexed by Carbon's platform to
-use in the website and other services.
+<PageDescription>
 
-Schemas are applied through metadata files written in YAML. If you're new to YAML and want to learn
-more, see "[Learn YAML in Y minutes](https://learnxinyminutes.com/docs/yaml)."
+Carbon resources (e.g. libraries, assets, design kits) follow a schema to standardize definitions
+and documentation. This structured information is indexed by Carbon's platform to use in the website
+and other services.
 
-**Example**
+</PageDescription>
 
-carbon.yml
+<AnchorLinks>
 
-```yml
+<AnchorLink>Stable version</AnchorLink>
```

Comment:
I'd suggest putting the yml example underneath the install code snippet for the stable schema. Otherwise it feels like the example should have an anchor link.

Reply:
I'm also wondering if this header should be 'Get started' -- something actionable.